WarOOP.Tests/ArmyTests.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections.Generic;
+using WarOOP.Models;
+using Xunit;
+
+namespace Tests;
+
+public class ArmyTests
+{
+    public static IEnumerable<object[]> TestData1 =>
+        new List<object[]>
+        {
+            new object[] { "Warrior", 15, "Knight", 7 },
+            new object[] { "Knight", 10, "Knight", 7 },
+            new object[] { "Knight", 5, "Warrior", 7 },
+            new object[] { "Warrior", 10, "Warrior", 7 }
+        };
+
+    public static IEnumerable<object[]> TestData2 =>
+        new List<object[]>
+        {
+            new object[] { "Warrior", 5, "Knight", 10 },
+            new object[] { "Knight", 10, "Knight", 12 },
+            new object[] { "Knight", 5, "Warrior", 9 },
+            new object[] { "Warrior", 10, "Warrior", 11 }
+        };
+
+    [Theory]
+    [MemberData(nameof(TestData1))]
+    public void FirstWinSecond(string armyType1, int armyCount1, string armyType2, int armyCount2)
```

Comment:
We agreed on other naming convention for tests.
Your tests don't cover edge-case scenarios. What if there is no units in army? Or you try to create units of wrong type?